pom.xml

```diff
@@ -153,12 +153,6 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>pipeline-stage-step</artifactId>
```

Comment:
🐛  you'll still need a dependency on pipeline-stage-step 2.2 for the example to actually run.